src/utils/helper-functions/utils.ts

```diff
@@ -10,3 +10,21 @@ export function findClassDeclarationNode(
   if (!node.parent) return null
   return findClassDeclarationNode(node.parent)
 }
+
+export function findImportDeclaration(
+  node: TSESTree.ClassDeclaration,
+  module: string,
+): TSESTree.ImportDeclaration | undefined {
+  let parentNode: TSESTree.Node | undefined = node
+
+  while ((parentNode = parentNode.parent)) {
+    if (parentNode.type !== 'Program') continue
+
+    return parentNode.body.find(
+      (node) => isImportDeclaration(node) && node.source.value === module,
+    ) as TSESTree.ImportDeclaration | undefined
+  }
+
+  // TODO: Remove this when enable tsConfig strict option.
+  return parentNode as TSESTree.ImportDeclaration | undefined
```

Comment:
When I was implementing this, I noticed that `strict` is not enabled. I could send a follow-up PR if desired.

Reply:
Good catch! Feel free to enable strict mode

Reply:
Done, #70.

Btw, thanks for adding me as a collaborator. It's a pleasure to help in this project :)

Reply:
We have to thank you 🤗